### ISSUE TITLE
View Rename_Clicking inside the formula line for floor plans crashed Revit 2019.2

### ISSUE THREAD

**Issue by MichaelVargg** [CyrilWaechter/pyRevitMEP#26]
When I tried out renaming my views  with your tool, I could not edit the formula for the name generation. All I did was click into the given values. When no cursor appeared, I clicked again. I did not count the number of times I clicked but I ended up with a Revit crash screen sending my data off to Autodesk. I only tried out floor plans so the other entries may be fine.

**Comment by MichaelVargg** [comment]
What did work, was saving the setup to the pyrevit-ini-file and editing that. After I did that, clicking into the formula row inside the application worked fine. Perhaps the crash had something to do with me using a different Revit localisation so that the original sample formula could only have worked in a French environment.

**Comment by CyrilWaechter** [comment]
Hi,

I launched Revit 2019.2 in english (ENU) and tried to reproduce you issue with no success. If parameter is not found, it uses parameter name instead. I do not understand why you cannot edit the formula. I was able to.

**Comment by MichaelVargg** [comment]
Hi Cyril,

now that I have changed or rather pre-edited the formula in pyrevit’s ini-file I cannot reproduce the problem anymore. My localization was a German one, which may have a different way of dealing with accented characters than the French or English versions. Thanks for looking into this. Is there some kind of log file that pyRevitMEP writes while its scripts are running that I could send you?

Best wishes,
Michael

Von: Cyril Waechter <notifications@github.com>
Gesendet: Freitag, 5. März 2021 15:31
An: CyrilWaechter/pyRevitMEP <pyRevitMEP@noreply.github.com>
Cc: Wolff Michael, ASSMANN BERATEN + PLANEN <mi.wolff@assmann.info>; Author <author@noreply.github.com>
Betreff: Re: [CyrilWaechter/pyRevitMEP] View Rename_Clicking inside the formula line for floor plans crashed Revit 2019.2 (#26)


Hi,

I launched Revit 2019.2 in english (ENU) and tried to reproduce you issue with no success. If parameter is not found, it uses parameter name instead. I do not understand why you cannot edit the formula. I was able to.
